### ISSUE TITLE
Deleted wallet on my mac recovery posible but no CraveNG folder help!

### ISSUE THREAD

**Issue by bitminer2017** [CooleRRSA/crave-ng#9]
I deleted my crave wallet from my Mac, tried reinstalling the wallet from the website but now when i try to run it it wont start I get this message

 Last login: Sun Feb  4 20:10:45 on ttys000
julios-imac:~ julio$ /Users/julio/Desktop/crave/Crave-2.5.0.1/bin/crave-qt ; exit;

************************
EXCEPTION: N5boost12interprocess22interprocess_exceptionE       
No such file or directory       
crave in Runaway exception
Please help me.

**Comment by bitminer2017** [comment]
![file](https://user-images.githubusercontent.com/34286479/35785236-f76a20ca-09ec-11e8-9c07-bf5ac44d4fbf.jpg)

**Comment by CooleRRSA** [comment]
This error means that crave-nq do not have permissions to write to data directory. Change owner of crave-ng executable.

**Comment by bitminer2017** [comment]
Thank You for the Respond, I believe i deleted the File CraveNG when i was trying to clean up my Computer. is there another way to create a new CraveNG as rel downloading the wallet and reinstalling it does not work. I tried.

**Comment by bitminer2017** [comment]
I do have possession
![back1](https://user-images.githubusercontent.com/34286479/36056904-81d62cd0-0dd6-11e8-8ad3-51453e7f76ed.jpg)
![back2](https://user-images.githubusercontent.com/34286479/36056907-852425d6-0dd6-11e8-8276-0d27d60c435e.jpg)

 of these files on my external HD

**Comment by CooleRRSA** [comment]
Try to run crave-qt wallet with -choosedatadir option.

**Comment by bitminer2017** [comment]
I updated My Mac to Sierrahigh and reloaded the Wallet, because it was a fresh wallet install I just had to transfer My back up folder "wallet.dat" to the new wallet Backups folder and it works Fine Now. Thanks for the support!

**Comment by NYCrypto1** [comment]
Hey bitminer,

I am having the same problem. Do you have any suggestions on how to transfer the backup wallet.dat file to a Ma that already is running HS? I still cant get my wallet to work and  Im running out of options.

**Comment by CooleRRSA** [comment]
@NYCrypto1 Try to redownload latest wallet https://github.com/CooleRRSA/crave-ng/releases/download/v2.5.0.2/crave-v2.5.0.2-macos64-highsierra.dmg
I have updated this build.

**Comment by NYCrypto1** [comment]
I am willing to give 250 CraveNG to anyone that can help me recover my old wallet.dat file and load it on the newest wallet version. Just hit me up.

**Comment by NYCrypto1** [comment]
I still don't understand why this dam wallet will not get passed loading the block index when the wallet first loads. If I had this wallet on my Windows PC no problem.. Thats what I get for trying to use dam MAC OS for staking!!

**Comment by CooleRRSA** [comment]
Dump your wallet private keys to text file at Windows PC wallet by "dumpwallet" command.
After this, move this file to the Mac and import your keys from text file by "importwallet" command.

**Comment by bitminer2017** [comment]
NYCrypto are You using The latest Version of Mac OX, SierraHigh?

**Comment by NYCrypto1** [comment]
yes.  but i still cant load any wallet , the old or the new, so I cant do the file transfer. something is stopping the wallet from loading.. just don't understand why. I can load other Mac OS wallets from other coins but it won't let me get past loading the block index as soon as the application opens

**Comment by bitminer2017** [comment]
My issue was that I deleted my wallet directly from my Applications list on my mac , did you do that mistake too?

**Comment by bitminer2017** [comment]
This is what I did and it worked to get my wallet back on track.

**Comment by bitminer2017** [comment]
![crave1](https://user-images.githubusercontent.com/34286479/36127580-42feef30-102c-11e8-8fc0-9577f3ffa7dd.jpeg)

**Comment by bitminer2017** [comment]
rhis should be there if You downloaded the New wallet from Crave website. Latest software on Your Mack must be Sierra.

**Comment by bitminer2017** [comment]
Keep clicking thru until you reach the folder that says Backups, that should be tthe last folder You select and it will be empty. it was in my case because i never could get my wallet to open so it never SYNC.


![crave2 back up](https://user-images.githubusercontent.com/34286479/36127658-8a538774-102c-11e8-9ae2-b346440acf94.jpeg)

**Comment by bitminer2017** [comment]
Open a new search window and locate your old wallet, look for the Backups folder like you did in the New wallet . You will find The Backups folder with Files in it.

![crave3 move backup](https://user-images.githubusercontent.com/34286479/36127809-396d6bda-102d-11e8-87b5-faeb454c70af.jpeg)

**Comment by bitminer2017** [comment]
Copy all the Files and Paste them into the (new ) Crave Backups Folder. Close everything and restart the Crave Wallet if it starts up , let it sync. Everything You had should show up. Good luck I hope this really works out for You. Please if it works Post it so that anyone with this same problem, can fix it.

**Comment by NYCrypto1** [comment]
I will attempt all of the suggestions tomorrow. Thank you for all of the responses. I hope that I will be able to apply and execute those fixes. This is why I invested in this coin, your support is A++  

    On Monday, 12 February 2018, 19:42, Mierx2017 <notifications@github.com> wrote:
 

 Keep clicking thru until you reach the folder that says Backups, that should be tthe last folder You select and it will be empty. it was in my case because i never could get my wallet to open so it never SYNC.—

**Comment by NYCrypto1** [comment]
great detailed explaination Bitminer, thank you. I am attempting your suggestions now. If I can succeed, I will let you know and I will send you 250 CraveNG...

**Comment by bitminer2017** [comment]
Your Very welcome NYCrypto1.

**Comment by NYCrypto1** [comment]
Ok, here is where I stand. I created a new user on my Mac and downloaded the new wallet from the website with no problem. Now I have a successfully downloaded wallet. I followed your instructions and put the wallet.dat file in the backup folder in CraveNG, but the balance is 0. That means I didn't tranfer the wallet.dat file correctly, but I see it in the backup file folder I put it. Do I erase the new wallet.dat backup file and just replace it with the old?

**Comment by CooleRRSA** [comment]
@NYCrypto1 It's better to export your private keys from working wallet and import them to new wallet.
I have described process above.

**Comment by NYCrypto1** [comment]
Ok, but the old wallet will not open, only the new wallet is open. I am afraid to download the old version after the new one successfully loaded.

**Comment by NYCrypto1** [comment]
Made new user and old wallet is loading.. cross my fingers that I can see my balance after I include old wallet.dat file

**Comment by CooleRRSA** [comment]
@NYCrypto1 Wait. You are trying to move wallet.dat from old Crave wallet v2.1.0.3 to new Crave wallet? v2.5.0.2?

**Comment by bitminer2017** [comment]
let it sync.Everything from the old wallet will show up.

**Comment by CooleRRSA** [comment]
You can't move coins from old Crave wallet to new. You must perform coinswap.
All info is posted at https://bitcointalk.org/index.php?topic=2547950

**Comment by bitminer2017** [comment]
@CooleRRSA I moved the .dat files from the old to the new and it work fine. I just had to sync wich took like 20-30 min.
@NYCrypto1 swap the wallet.data that are outside the folder.
![crave3 move backup](https://user-images.githubusercontent.com/34286479/36226961-768e4a2c-119d-11e8-87cf-e507ada73c63.jpeg)

**Comment by NYCrypto1** [comment]
@CooleRRSA,
No I already did the coin swap. I am speaking about the wallet version before the latest.
I am almost there I believe. I just need to change the wallet.dat files outside the backups wallet and see what happens.

Once again thanks

**Comment by NYCrypto1** [comment]
OMG!!!! I DID IT.. LOL...WHOOOOO HOOOOOOO!!!! 

you have no idea what a headache that was for me to complete, but most likely a routine task for you guys. 
Bitminer, please reply with your wallet address. If it wasnt for those screenshots  I wouldnt know what to do.. It would be my pleasure to send you 250CraveNG coins. THANK YOU

**Comment by bitminer2017** [comment]
Your Very Welcome WooHooo !!! I'm Happy For You My Friend.

VTsDS4eQvohFQ4hzN1UUUYo3pLPKQqXKiX

This is My wallet address.

**Comment by NYCrypto1** [comment]
Just transfered 237 CraveNg to your wallet biminer2017. Sorry, I know I promised 250 but I need 5000 exactly.. BUT TYVM for you help!!

**Comment by bitminer2017** [comment]
Thank You for Being a man of His word It was a pleasure Helping You and Happy everything worked out! Now there is information out there that might just help many like us.
